Gemfile

```diff
@@ -1,13 +1,13 @@
 source 'https://rubygems.org'
 
-ruby '2.7.8'
+ruby '3.0.0'
 gem 'puma', '4.3.12'
-gem 'rails', '~> 6.0'
+gem 'rails', '~> 6.1.0'
 gem 'pg', '~> 1.1'
 gem 'sass-rails'
 gem 'uglifier', '>= 1.3.0'
 
-gem 'bigdecimal', '1.3.5'
+gem 'bigdecimal', '2.0.0'
```

Comment:
good question. I looked into it and i don't think we need it. i removed it on the 7.1.2 branch and the tests passed, and i deployed to production and it seems like it's fine.

according to chatgpt:

> The bigdecimal gem is part of the Ruby standard library and provides a BigDecimal class for arbitrary-precision decimal arithmetic. This class is useful when working with numbers that require high precision, particularly in financial applications or any scenario where accurate decimal calculations are crucial.

> If you're not using or planning to use the BigDecimal class in your application, you may not need to explicitly include the bigdecimal gem in your Gemfile. In most cases, you can safely remove it, and your application will continue to work fine

So I'll keep it in this PR since there are other branches branched off of this one, and i'll remove it in the 7.1.2 branch